### ISSUE TITLE
Toggle between Calendar and Episode list views

### ISSUE THREAD

**Issue by brandonhundt** [PRX/publish.prx.org#654]
There isn't a way to go back to the Episode list from the calendar.

**Comment by brandonhundt** [comment]
Bumping from this sprint. Blocked by ongoing UX interviews in #646 .

**Comment by brandonhundt** [comment]
closing in favor of #655
